src/third-party/README.md

```diff
@@ -0,0 +1,2 @@
+These are packages that we depend upon. They may need to be trimmed, all .cpp
```

Comment:
I considered that too but from what I gather symlinks in git don't work very well on windows machines.